.travis.yml

```diff
@@ -4,6 +4,12 @@ scala:
   - 2.11.11
 jdk:
   - oraclejdk8
+  - oraclejdk9
+
+matrix:
+  exclude:
+    - scala: 2.11.11
+    - jdk: oraclejdk9
```

Comment:
Looks like Travis now only tests a single combination: scala 2.12 + java 8

I think the right config would be (only one list element):

```yaml
matrix:
  exclude:
    - scala: 2.11.11
      jdk: oraclejdk9
```